src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,42 @@
+package duke.task;
+
+/**
+ * Represents a Task without a specific date/time.
+ * @author Aaron Pang
+ * @version CS2103T AY22/23 Semester 1
+ */
+public class Todo extends Task {
+    /**
+     * Constructs a to-do task without isDone.
+     *
+     * @param description Description of the to-do task.
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Construst a to-do task with isDone.
+     * @param description Description of the to-do task.
+     * @param isDone True if task is done, false otherwise
+     */
+    public Todo(String description, boolean isDone) {
```

Comment:
I liked that you do not "trip up" readers, you have no unused parameters in the method signature and all your variables are unique. This follows the code quality guidelines. Good Job!